src/query/settings/src/lib.rs

```diff
@@ -471,6 +471,16 @@ impl Settings {
                 desc: "How many hours will the COPY file metadata expired in the metasrv, default value: 24*7=7days",
                 possible_values: None,
             },
+            SettingValue {
+                default_value: UserSettingValue::UInt64(0),
+                user_setting: UserSetting::create(
+                    "insert_values_disable_expression",
```

Comment:
Let's introduce a new setting: `insert_values_enable_expression`, default is false. in most cases, there is no expression, so the user no need to change anything they can use it.

Reply:
make sense in production,
but for now, many value need to be expressed with the expression? like `parse_json`?
some  logic tests may be affected?

Reply:
Not sure if `insert_values_enable_expression=false`, if the insert value has an expression, can we return an error let the user `enable insert_values_enable_expression`?

Reply:
> Let's introduce a new setting: insert_values_enable_expression, default is false.

It may take a bad effect on users because it will not be compatible with the script which already works fine.

Reply:
It should be ok, bulk insert mostly has no expressions, it is the raw sql that is dumped by mysqldump, are there any other cases?
The key point here is that: if he has expression in the bulk insert, we give an error hint to let him enable expression on.

Reply:
We can make the default value of `insert_values_enable_expression` as true, this is compatible with the current logic, and users can choose to turn it off if they want better performance.

Reply:
Ok.